CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+
+## Doubts solving
+
+:two_women_holding_hands: To help less advanced users is another wonderful way to start. You can help us to close some open issues. This kind of ticket should be labeled with `question`.
+
+## Improvement proposal
+
+:smiling_imp: If you have an idea for a new feature please open a ticket labeled as `enhancement`. If you could also add a piece of code with the idea or a partial implementation it would be awesome.
+
+## Documentation
+
+:eyes: We all know the doc always needs fixes/upgrades :wink:, so please feel free to send a PR (see next point) with what you found.
+
+## Code
+
+:star: This section includes some tips that will help you to push source code.
+
+### Commits
+
+Please follow the next rules for the commit messages:
+
+- It should be formed by a one-line subject, followed by one line of white space. Followed by one or more descriptive paragraphs, each separated by one line of white space. All of them finished by a dot.
+- If it fixes an issue, it should include a reference to the issue ID in the first line of the commit.
+- It should provide enough information for a reviewer to understand the changes and their relation to the rest of the code.
+
+### Pull requests
+
+- We use [GitHub pull requests](https://help.github.com/articles/about-pull-requests) to accept the contributions.
+- Except for proposals (see next point), please, use [pull requests](https://github.com/Qiskit/openqasm/pulls) as is to submit a new one :smile:.
+- Review the parts of the documentation regarding the new changes and update it if it's needed.
+- New features often imply changes in the existent tests or new ones are needed. Once they're updated/added please be sure they keep passing.
+
+## Spec proposals
+
+:bulb: All new ideas go through the next stages to become a new feature of the language.
+
+- Draft: Backlog items with different levels of abstraction. Anybody can add one issue in the [main repo](https://github.com/Qiskit/openqasm). Please label it as `draft`.
+- Proposal: An idea with the correct form:
+  - Add one issue in the [main repo](https://github.com/Qiskit/openqasm) labeled as `proposal` using [this template](templates/proposal.md).
+- Candidate: During each monthly meeting the assistants select the ones considered more interesting to pass to the next stage. One of the core devs will start commenting on the issue to guide the owner into the next steps, including:
+  - Fork [the main repo](https://github.com/Qiskit/openqasm).
+  - Add the content of the proposal, note that conformance tests are mandatory at this point.
+  - Make a pull request.
+  - The core dev can ask for changes before reaching the next stage.
+- Accepted: When the PR is merged into main.
+
+## Tests
```

Comment:
This section looks copied from qiskit and is not correct for this repo's tests.

Reply:
Fixed in 4e8bb1e.